services/Makefile.am

```diff
@@ -0,0 +1,85 @@
+ACLOCAL_AMFLAGS = ${ACLOCAL_FLAGS} -I m4
+
+lib_LTLIBRARIES = libbm_grpc_dataplane.la
+
+EXTRA_DIST = p4/bm/dataplane_interface.proto \
+switch_runner/switch_runner.cpp \
+switch_runner/switch_runner.h
```

Comment:
Am I missing something, that doesn't look right?

Reply:
Thank you for catching this. I have another commit that moves `switch_runner` from `targets/simple_switch_grpc/` to `services/` to make it reusable with other targets. I will include this commit in the next pull request.